pkg/controller/container-runtime-config/helpers.go

```diff
@@ -168,14 +172,64 @@ func findPolicyJSON(mc *mcfgv1.MachineConfig) (*igntypes.File, error) {
 	return nil, fmt.Errorf("could not find Policy JSON")
 }
 
+func getManagedKey(pool *mcfgv1.MachineConfigPool, suffix string) string {
+	return fmt.Sprintf("99-%s-%s", pool.Name, suffix)
+}
+
+// Deprecated: use getStableManagedKeyCtrCfg
```

Comment:
Should we add a warning log on this so it's caught when called? If so, I the actual functionality in this function is small enough to be used in `getStableManagedKeyCtrCfg` directly.

Reply:
if you're using Code it'll tell you that it's deprecated when referring to it :) as for adding logs in the running cluster, I think we as code maintainers we should know the code well enough (if we're doing reviews) to catch that something it's old and deprecated. So I'm gonna do something and just rename `getManagedKeyCtrCfg` to `getManagedKeyCtrCfgDeprecated` as it's still used in unit to make sure the upgrade that we're doing in this code still works.